client.go

```diff
@@ -97,14 +97,43 @@ func (c *Client) post(endpoint string, body interface{}) ([]byte, error) {
 	return buf, nil
 }
 
+func (c *Client) stream(endpoint string, body interface{}) (io.ReadCloser, error) {
```

Comment:
isn't body always of type `GenerationOptions`?

Reply:
no, that's the other `Stream()`. I'm thinking if I should just make this strongly typed too.

Reply:
nvm, you are right, i thought we had dedicated request type, we don't